nixos/modules/epics-environment.nix

```diff
@@ -51,12 +51,113 @@ in
       type = with lib.types; listOf str;
       default = [ ];
     };
+    openCAFirewall = lib.mkOption {
+      description = ''
+        Open the default ports of the Channel Access protocol.
+        Enabling this option makes IOCs reachable using unicast addresses.
+        To make it discoverable with the auto address list / broadcast addresses,
+        use the {option}`allowCABroadcastDiscovery` option.
+        :::{note}
+        If you changed the Channel Access ports in your IOC,
+        open these ports in the firewall manually
+        by using `networking.firewall.allowedTCPPorts`
+        and `networking.firewall.allowedUDPPorts`.
+        :::
+      '';
+      type = lib.types.bool;
+      default = false;
+    };
+
+    openPVAFirewall = lib.mkOption {
+      description = ''
+        Open the default ports of the pvAccess protocol.
+        Enabling this option makes IOCs reachable using unicast addresses.
+        To make it discoverable with the auto address list / broadcast addresses,
+        use the {option}`allowPVABroadcastDiscovery` option.
+        :::{important}
+        This option currently only work with PVXS IOCs,
+        and doesn't work with epics-base's pvAccess
+        :::
+        :::{warning}
+        This opens the firewall on all network interfaces.
+        :::
+        :::{note}
+        If you changed the pvAccess ports in your IOC,
+        open these ports in the firewall manually
+        by using `networking.firewall.allowedTCPPorts`
+        and `networking.firewall.allowedUDPPorts`.
+        :::
+      '';
+      type = lib.types.bool;
+      default = false;
+    };
+    allowCABroadcastDiscovery = lib.mkOption {
+      description = ''
+        This option allows the network discovery of Channel Access IOCs on the default port.
+        :::{danger}
+        This option is an security issue
+        and attackers crafting a malicious packet from source port 5064
+        will be able to access any [Ephemeral port]
+        of this machine.
+        :::
+        :::{warning}
+        This opens the firewall on all network interfaces.
+        :::
+          [Ephemeral port]: https://en.wikipedia.org/wiki/Ephemeral_port
```

Comment:
```suggestion
```

Else there's a warning during the docs build that the "Ephemeral port" reference is duplicated.